tox.ini

```
@@ -0,0 +1,43 @@
+[tox]
+envlist =
+    django4-{python3.8,python3.9,python3.10,python3.11,python3.12},
+    django5-{python3.10,python3.11,python3.12},
+    # run one of the tests again but with coverage
+    coveralls-django4-python3.11,
```

Should we use the latest PY version for the coveralls test? cc: @farridav 
```suggestion
    coveralls-django4-python3.12,
```